src/lib/users/clients/auto.rb

```diff
@@ -105,7 +112,7 @@ def modified
       end
 
       def reset
-        Y2Users::ConfigManager.instance.unregister(:autoinst)
+        import({})
```

Comment:
np: the result might be different, although I do not think it is a problem. When calling reset, you will have an empty configuration at :autoinst, or am I missing something? Is it relevant? I do not think so, but perhaps it is not the expected result of calling `#reset`.

Reply:
this is basically revert to previous version as reset is called only in autoyast UI. So I keep it as broken as it was before.

Reply:
so `:autoinst` case should not happen as it is `config` mode.

Reply:
OK, fine with me. I was simply wondering if it was correct from a logic point of view.